assets/src/components/PostMeta/withPostMeta.js

```diff
@@ -59,17 +51,8 @@ export function withPostMeta( WrappedComponent ) {
       }
     }
 
-    componentDidUpdate() {
-      const { metaKey, postMeta } = this.props;
-      const metaValue = postMeta[ metaKey ];
-
-      if (metaValue !== this.state.value) {
-        this.handleChange( metaKey, this.state.value );
-      }
-    }
-
     render() {
-      const { metaKey, postMeta, writeMeta, onChange, ...ownProps } = this.props;
+      const { metaKey, postMeta, writeMeta, getNewMeta, onChange, defaultValue, ...ownProps } = this.props;
```

Comment:
Added this `defaultValue` props so that it's not in `ownProps`. That was causing issues as React uses the same prop name itself and was causing following warning: `Warning: Select elements must be either controlled or uncontrolled (specify either the value prop, or the defaultValue prop, but not both). Decide between using a controlled or uncontrolled select element and remove one of these props. More info: https://fb.me/react-controlled-components`

Reply:
Right, I saw that warning, I think this change will cause the `withDefaultLabel` HOC to stop working as it relies on the `defaultValue` prop, maybe we can use a different name for the prop, I tried that but it causes an "Unknown Prop Warning", also quite common. So maybe we need to review how that works and make sure we don't use `defaultValue` at all anywhere.

Reply:
Possible hack: React has a feature where using a non-standard lowercase prop (e.g.: `themedefault`) won't trigger the "Unknown prop in DOM component" warning. But we probably want to avoid that also.

Reply:
Yeah I re-added it to not break `withDefaultLabel`. The "Unknown Prop Warning" is because the prop isn't being removed with destructuring, but called with `this.props.defaultValue`. So `ownProps` is passed with an unknown attribute to react. If we rename it both issues are fixed.